std_msgs/ColorRGBA   color       #sequence color
float32              hold_time   #time how long the color should be hold [s]
float32              cross_time  #time how long it take to fade to this color [s]